cmd/search/main.go

```diff
@@ -122,16 +137,26 @@ type options struct {
 	jobsPath     string
 	jobURIPrefix *url.URL
 
-	bugs         *bugzilla.CommentStore
-	bugsPath     string
+	bugs *bugzilla.CommentStore
+	//jira
+	issues *jira.CommentStore
+
+	bugsPath string
+	//jira
+	issuesPath string
+
 	bugURIPrefix *url.URL
+	//jira
+	issueURIPrefix *url.URL
```

Comment:
I would group the individual pieces together.  Then you dont have `//jira` comments every couple lines.

Reply:
Done